trace/cloud-trace-demo-app/app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6.0
+FROM python:3.8.0b1
```

Comment:
@ZiweiZhao @dukeng what do you think about this auto-update?

Reply:
LGTM :) I don't think the Python version will effect much to the demo app.